app/templates/components/slide-modal.hbs

```diff
@@ -0,0 +1,13 @@
+<div class = "add-employee-slide-container font-style">
+  <div class = "add-employee-outerslide"></div>
+  <div class = "add-employee-slide">
+    <div class = "header slideheader">
+      <div class="slide-headertext">Add Employee</div>
+      <div class="add-employee-slide-btns">
+        <div class="apply-btn">Apply</div>
+        <div class="reset-btn">Reset</div>
```

Comment:
We can write this component as a common wrapper and send necessary parameters and yield to fit for the add employee functionality. This way, the slide model can be reused for any other module.